frontend/src/queries/schema.ts

```diff
@@ -930,13 +930,15 @@ export interface WebStatsTableQuery extends WebAnalyticsQueryBase {
     response?: WebStatsTableQueryResponse
     includeScrollDepth?: boolean // automatically sets includeBounceRate to true
     includeBounceRate?: boolean
+    limit?: number
```

Comment:
This needs a `@asType integer` to become integer in Python, see the other limits.